micrometer-registry-signalfx/src/main/java/io/micrometer/signalfx/SignalfxDistributionSummary.java

```diff
@@ -61,11 +62,14 @@ final class SignalfxDistributionSummary extends AbstractDistributionSummary {
 
     private final TimeWindowMax max;
 
+    private final DeltaHistogramSnapshot deltaHistogramSnapshot;
+
     SignalfxDistributionSummary(Id id, Clock clock, DistributionStatisticConfig distributionStatisticConfig,
-            double scale, long stepMillis) {
+            double scale, long stepMillis, boolean isDelta) {
         super(id, clock, CumulativeHistogramConfigUtil.updateConfig(distributionStatisticConfig), scale, false);
         this.countTotal = new StepTuple2<>(clock, stepMillis, 0L, 0.0, count::sumThenReset, total::sumThenReset);
-        max = new TimeWindowMax(clock, distributionStatisticConfig);
+        this.max = new TimeWindowMax(clock, distributionStatisticConfig);
+        this.deltaHistogramSnapshot = new DeltaHistogramSnapshot(isDelta);
```

Comment:
Same comment about passing in the actual dependency instead of creating an instance here in the constructor.

Reply:
See previous response. I will resolve this comment and continue the discussion there, if we decide something will make sure will apply here as well.